platform/bitcoin/api_test.go

```diff
@@ -126,16 +146,41 @@ func TestInferDirection(t *testing.T) {
 		t.Error("direction is not outgoing")
 	}
 
-	inputs2 := []string{
-		"3CgvDkzcJ7yMZe75jNBem6Bj6nkMAWwMEf",
-		"3LyzYcB54pm9EAMmzXpFfb1kzEDAFvqBgT",
-		"3Q6DYour5q5WdMhyXsyPgBeAqPCXchzCsF",
-		"3JZZM1rwst7G5izxbFL7KNvy7ZiZ47SVqG",
+	inputs2 := []blockatlas.TxOutput{
+		blockatlas.TxOutput{
```

Comment:
The same applies for all object in this array. We don't need to define the type again.

```suggestion
		{
```